pkg/gen/python-templates/kind.py.mustache

```diff
@@ -1,9 +1,11 @@
 import pulumi
 import pulumi.runtime
 
+from ...tables import _CASING_FORWARD_TABLE, _CASING_BACKWARD_TABLE
```

Comment:
Not a huge deal, but it seems like it would be nicer to have this read `tables._CASING_FORWARD` instead.